test/salt/top.sls

```diff
@@ -1,6 +1,4 @@
 base:
   '*':
-    - postgres.client
-    - postgres
```

Comment:
we should still enable the postgres "upstream" repository so that cubicweb-saem_ref's RPM can pull the proper postgres-contrib package.